src/features/defi/providers/fox-farming/hooks/useFoxFarming.ts

```diff
@@ -236,38 +293,66 @@ export const useFoxFarming = (
     const txid = await buildAndBroadcast({
       adapter,
       buildCustomTxInput: {
+        from: accountAddress,
         accountNumber,
         to: uniV2LPContract.address,
         value: '0',
         data,
-        wallet,
         ...fees,
       },
     })
 
     return txid
-  }, [accountNumber, adapter, ethAsset.chainId, contractAddress, getApproveFees, wallet])
+  }, [
+    accountNumber,
+    accountAddress,
+    adapter,
+    ethAsset.chainId,
+    contractAddress,
+    getApproveFees,
+    wallet,
+  ])
 
   const claimRewards = useCallback(async () => {
-    if (skip || !isValidAccountNumber(accountNumber) || !wallet) return
+    if (skip || !isValidAccountNumber(accountNumber) || !wallet || !accountAddress) return
 
     if (!adapter) throw new Error(`no adapter available for ${ethAsset.chainId}`)
 
     const data = foxFarmingContract.interface.encodeFunctionData('getReward')
 
+    const fees = await getFees({
+      from: accountAddress,
+      adapter,
+      data,
+      to: contractAddress,
+      value: '0',
+      supportsEIP1559,
+    })
+
     const buildCustomTxInput = await createBuildCustomTxInput({
       accountNumber,
       adapter,
       data,
       to: contractAddress,
       value: '0',
       wallet,
+      chainSpecific: fees,
```

Comment:
Ideally exclude fees from `chainSpecific` required property, since createBuildCustomTxInput does the same check